java-checks/src/main/java/org/sonar/java/checks/CheckList.java

```diff
@@ -451,6 +451,7 @@ private CheckList() {
       .add(FilesExistsJDK8Check.class)
       .add(StaticImportCountCheck.class)
       .add(ClassFieldCountCheck.class)
+      .add(DisallowedClassCheck.class)
```

Comment:
I'm not sure you rebased correctly your branch. Are you sure the master branch you are looking at is up to date?

The conflict comes from this line. In our master branch, there is 4 more check after `ClassFieldCountCheck.class`.
See here: [CheckList.java#L457](https://github.com/SonarSource/sonar-java/blob/4.3-RC1/java-checks/src/main/java/org/sonar/java/checks/CheckList.java#L457)